src/docs/product/profiling/getting-started.mdx

```diff
@@ -24,6 +24,7 @@ Profiling depends on Sentry's performance monitoring product being enabled befor
   - [React Native](/platforms/react-native/profiling/) [beta]
   - [Flutter](/platforms/flutter/profiling/) [experimental, iOS and macOS only]
```

Comment:
We could add ".NET (On iOS for Native and MAUI)" here too

Reply:
```suggestion
  - [Flutter](/platforms/flutter/profiling/) [experimental, iOS and macOS only]
  - [.NET](/platforms/dotnet/profiling/) [experimental, iOS only]
```